great-idea/index.html

```diff
@@ -14,10 +14,70 @@
   <![endif]-->
 </head>
 
-  <body>
-
-    <!-- Copy and paste your HTML from the first UI project here -->
-      
-    
+<body>
+    <header>
+      <div class="top">
+        <nav>
+            <a href="#">Services</a>
```

Comment:
To link one page to another, simply write in the HTML or URL inside the "", for example, <a href="services.html">. That will make the link clickable and take the user to the services.html